backend/app.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+
+app = Flask(__name__)
+api = Api(app)
+
+def Response(result, error):
+    return { 'result': result, 'error': error }
+
+class SignIn(Resource):
+    def post(self):
+        email = request.form['email']
+        password = request.form['password']
+        return Response(None, '{"Not implemented"}')
+
+
+class SignUp(Resource):
+    def post(self):
+        email = request.form['email']
+        password = request.form['password']
+        password2 = request.form['password2']
+        return Response(None, '{"Not implemented"}')
+
+
+class Posts(Resource):
+    def get(self):
+        limit = request.form['limit']
+        offset = request.form['offset']
+
+        return Response(None, '{"Not implemented"}')
+    
+    def post(self):
+        xtoken = request.headers['xtoken']
```

Comment:
Headers fields are case insensitive, ok.
https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive#5259004

Here more about `x-` headers:
https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions#3561399 

To avoid using `x-` header please change it to `Authorization`
```
headers['Authorization']
```

We will contain token in `Authorization` header